ding_pretix.module

```diff
@@ -193,6 +203,9 @@ function ding_pretix_node_update($node) {
 
 /**
  * Implements hook_node_delete().
+ *
+ * @param $node
+ *   The node that is being deleted.
```

Comment:
Hooks used short form comments normally: https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#hookimpl

Reply:
Please don't kill the messenger....